### PR TITLE
Restructure the airqlouds dropdown

### PR DESCRIPTION
**Pull request by danielmarv**
SelectorModal

#### Summary of Changes (What does this PR do?)

- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] I've updated corresponding documentation for the changes in this PR.

#### How should this be manually tested?

- Instead of a dropdown input, i replaced with a text field which text content changes according to the variant 
corresponding city or country selected.
- Onclick, it pops up a Modal with all the data and accepts the interactivity

#### What are the relevant tickets?

- [AN-422](https://airqoteam.atlassian.net/browse/AN-422)

#### Screenshots (optional)

- Now this is how the dropdown looks like;

- When the user clicks on the dropdown, it drops out this list which is interactive.

- Now a user is able to search for airqlouds

[AN-422]: https://airqoteam.atlassian.net/browse/AN-422?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by FaithDaka** [comment]
@Baalmart What is meant to be the final output of this task? I can't seem to see what exactly is being achieved.

**Comment by Baalmart** [comment]
> @Baalmart What is meant to be the final output of this task? I can't seem to see what exactly is being achieved.

Thanks @FaithDaka , so the final output is to have some "smart" way of selecting AirQlouds on the AirQo dashboard.
That is, a smart way that will accommodate our ever increasing number of new AirQlouds. The current dropdown implementation is a bit chaotic to utilise for a huge list of AirQlouds. I have created a Github issue with more details here: https://github.com/airqo-platform/AirQo-frontend/issues/1225

**Review by Baalmart — DISMISSED** [review]
Thanks @danielmarv , thanks for the implementation.
All works well but something seems off, the look and feel has a significant room for improvement.

I am not sure if it the tool you are using or something but I encourage you to look further into how we can improve the look and feel since this is a very busy page of the platform. It is one of the most utilised pages.

**Review by Baalmart — COMMENTED** [review]
Hi @danielmarv , to make your life much easier, you could take advantage of https://react-select.com/home

**Review by NoahNsimbe — COMMENTED** [review]
thanks @danielmarv. There are a few PR comments you might want to address.

**Comment by danielmarv** [comment]
i think now its ok

File: netmanager/src/assets/css/dropdown.css
Comment (Baalmart) [inline netmanager/src/assets/css/dropdown.css:3]: Thanks @danielmarv , basing on the deploy preview link, all looks good to me now.

I request you to consult with your supervisor about the CSS changes in this file. Just want to make sure that the rest of the system remains intact.

Once you have alignment in form of an approval from your supervisor, the PR will be merged.

Thanks!

Comment (Codebmk) [reply netmanager/src/assets/css/dropdown.css:3]: Awesome work @danielmarv 
Some small feedback: Could we adjust the dropdown button's height to match that of the refresh button? Also, the font size of the items within the dropdown seems pretty small on my end, may be setting them to 18px..

image

**Review by Codebmk — APPROVED** [review]
LGTM thanks

File: netmanager/src/assets/css/dropdown.css
Comment (Codebmk) [inline netmanager/src/assets/css/dropdown.css:12]: @danielmarv if this commented code isn't needed, you can remove it

**Pull request merged**
